examples/GLGradientLegendItem.py

```diff
@@ -0,0 +1,44 @@
+import initExample
+
+from pyqtgraph.Qt import QtCore, QtGui
+import pyqtgraph as pg
+import pyqtgraph.opengl as gl
+import numpy
+
+app = pg.mkQApp()
+w = gl.GLViewWidget()
+w.show()
+w.setWindowTitle('pyqtgraph example: GLGradientLegendItem')
+w.setCameraPosition(distance=60)
+
+gx = gl.GLGridItem()
+gx.rotate(90, 0, 1, 0)
+w.addItem(gx)
+
+md = gl.MeshData.cylinder(rows=10, cols=20, radius=[5., 5], length=20.)
+md._vertexes[:,2] = md._vertexes[:,2]-10
+
+# set color based on z coordinates
+cmap = pg.ColorMap((0,.25,.5,.75,1),((0,0,255),(0,255,255),(0,255,0),(255,255,0),(255,0,0)))
+h = md.vertexes()[:,2]
+#remember these
+hmax, hmin = h.max(), h.min()
+h = h-hmin
+h = h/h.max()
+colors = cmap.map(h)/255.0
```

Comment:
Can use `map(h, mode="float")` and get rid of the division by 255 for clarity. I dunno, maybe that's not really that much clearer.